docs/Support.md

```diff
@@ -0,0 +1,38 @@
+# Support
+
+## I need help right now
+
+1. Don't panic
+2. Page MoFo on-call support with a clear description of the problem and a method to contact you (heading to #fundraising on irc.mozilla.org is a good idea)
+3. Try to gather information about the issue while waiting for on-call to acknowledge
+4. If on-call has not acknowledged your issue in 15 minutes then use the Mozilla Phonebook to start calling engineers and/or management that would be able to help
```

Comment:
i would put an order here, otherwise this sounds like carte blanche* to jsut start calling everyone willy nilly

* i barely know what this means / probably spelled it wrong